### ISSUE TITLE
Build Fails when build target set to px4esc_1_6

### ISSUE THREAD

**Issue by davids5** [thiemar/vectorcontrol#3]
```
/usr/lib64/ccache/arm-none-eabi-g++ -mthumb -mcpu=cortex-m4 -mfloat-abi=hard -mfpu=fpv4-sp-d16 -c -O3  -g3 -ffreestanding -fsingle-precision-constant -fno-common -fmessage-length=0 -Wall -fno-exceptions -ffunction-sections -fdata-sections -fno-rtti -fno-threadsafe-statics -DDEBUG -include config_px4esc_1_6.h -DTOOLCHAIN_GCC_ARM -DNO_RELOC='0' -DUAVCAN_TINY=1 -DUAVCAN_DEBUG=0 -DUAVCAN_GENERAL_PURPOSE_PLATFORM=0 -DUAVCAN_NOEXCEPT=1 -DUAVCAN_TOSTRING=0 -DUAVCAN_USE_EXTERNAL_SNPRINTF=1 -DUAVCAN_NO_ASSERTIONS=1 -DUAVCAN_NO_GLOBAL_DATA_TYPE_REGISTRY=1 -std=c++11 -Iapp/include/ -Iarch/include -Iarch/src -Iarch/src/chip -I/usr/lib64/ccache/../arm-none-eabi/include/ -Ilibuavcan/include -Ilibuavcan/include/dsdlc_generated -I/usr/lib64/ccache/../arm-none-eabi/include/  -o build/app/hal.o app/src/hal.cpp
In file included from arch/src/stm32_rcc.h:43:0,
                 from arch/src/stm32.h:85,
                 from ./config_px4esc_1_6.h:71,
                 from <command-line>:0:
app/src/hal_px4esc_1_6.cpp: In function 'void hal_read_phase_shunts_(int16_t*)':
app/src/hal_px4esc_1_6.cpp:187:55: error: 'STM32_ADC1_JDR1' was not declared in this scope
         phase_shunt_signal_lsb[1] = (int16_t)getreg32(STM32_ADC1_JDR1);
```

**Comment by bendyer** [comment]
Sorry, moved where the includes happen to work better with the new headers and forgot to update the PX4ESC HAL file. Fixed in 3e273032ca4843894aea445194b40d42b346a70e.

**Comment by davids5** [comment]
@bendyer Thank you! I builds fine with my PR.
